mobility_sensing/scripts/python_pointclouds.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
```

Comment:
Should this file really be in this commit?  If not, I'd do

git rm --cached <path/to/file>  in order to keep the file for a later commit, but to not include it in this PR